Assets/MRTK/Core/Inspectors/Setup/MixedRealityEditorSettings.cs

```diff
@@ -98,10 +98,17 @@ private static void LogConfigurationWarnings()
                 Debug.LogWarning("<b>Virtual reality supported</b> not enabled. Check <i>XR Settings</i> under <i>Player Settings</i>");
             }
 
+#if UNITY_ANDROID
+            if (!MixedRealityOptimizeUtils.IsSinglePass())
```

Comment:
Maybe use the same IsSinglePassInstanced, etc and move the UNITY_ANDROID checks inside those methods?